src/stache/includes/SkyAppTestModule-note.html

```diff
@@ -1 +1 @@
-Note that in the sample unit tests, the <stache-code>configureTestingModule</stache-code> function imports <stache-code>SkyAppTestModule</stache-code> for testing convenience. This module brings in your all of your SPA's components and modules. By default, SPAs import the entire SKY UX library, so <stache-code>SkyAppTestModule</stache-code> imports all components and modules into your tests. This can negatively impact test performance for applications that only use a subset of SKY UX components, but we still recommend that you import <stache-code>SkyAppTestModule</stache-code> into your <stache-code>TestBed</stache-code>. To optimize performance when tests don't require access to all SKY UX components and modules, use <a routerLink="/learn/reference/configuration">the <stache-code>skyuxModules</stache-code> configuration property</a> to specify the SKY UX modules to import into your application. This option provides a balance between performance and the simplicity that SKY UX Builder provides.
\ No newline at end of file
+Note that in the sample unit tests, the <stache-code>configureTestingModule</stache-code> function imports <stache-code>SkyAppTestModule</stache-code> for testing convenience. This module brings in your all of your SPA's components, modules defined in the <stache-code>app-extras.module.ts</stache-code> file, and locale resource strings added in your SPA's <stache-code>./src/assets/locales</stache-code> folder.
```

Comment:
I had trouble following the last sentence at first. Part of that was the extra "your" in "your all of your," but I also didn't realize that we were listing 3 things until I read this two or three times. I think it would be clearer if we tightened up a bit and added "all" (or "the") before the last 2 items: "This module brings in all of your SPA's components, all modules defined in `app-extras.module.ts`, and all locale resource strings in `./src/assets/locales`." What do you think?

Reply:
I like that! Done